sections/rust.zsh

```diff
@@ -19,19 +19,34 @@ SPACESHIP_RUST_COLOR="${SPACESHIP_RUST_COLOR="red"}"
 # ------------------------------------------------------------------------------
 
 # Show current version of Rust
-spaceship_rust() {
+spaceship_async_job_load_rust() {
   [[ $SPACESHIP_RUST_SHOW == false ]] && return
 
+    async_job spaceship spaceship_async_job_rust $PWD
+}
+
+spaceship_async_job_rust() {
   # If there are Rust-specific files in current directory
+  setopt extendedglob
```

Comment:
This probably is not (always?) working unless you add `builtin cd -q $1 2>/dev/null` as the first line in this function - I have #3 open for ruby as well, but in truth I'm waiting for https://github.com/mafredri/zsh-async/pull/29 to be merged, then these hacks with passing `$PWD` won't be necessary.